### DISCUSSION TITLE
Dagster GraphQL API cookbook

### DISCUSSION THREAD

**Discussion by greaber** [dagster-io/dagster#21241] [Q&A]
I've found that there are lots of things I want to do in dagster that aren't supported by the UI.  Using the dagster graphql API is probably a solution, but this API is super complicated, and I don't have time to spend days studying it.  Also, neither ChatGPT nor Scout is much help at creating valid API requests.  Is there somewhere some kind of cookbook that shows how to accomplish common tasks programmatically?  For instance, I want currently just to get a list of failed partitions of an asset.  This seems like a really simple thing, and you can even get such a list in the UI (but you can't export it or even copy it to the clipboard).  I spend some time trying to figure out how to do this in the API, but I didn't succeed right away, and I don't have time to put more effort into this.

**Comment by garethbrickman** [comment]
This is on our roadmap to improve 👍

**Comment by TCronino** [comment]
Would also be interested in this!
